contrib/android.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+set -x
+test x$NDK = x  && exit 1
+root="$(readlink -f $(dirname $0)/../)"
+out=$root/lokinet-android-$(git describe)
+mkdir -p $out
+mkdir -p $root/build-android
+cd $root/build-android
+for abi in armeabi-v7a arm64-v8a x86 x86_64 ; do
```

Comment:
prefer to define the ABI list in a separate line above so it's clear where one should go to edit that list if ever we need to.

Reply:
fixed in https://github.com/oxen-io/loki-network/pull/1673/commits/b7485b4b25bfab4033045bfaaa0f6a242a69b814